public/svgs/frost/list-large.svg

```diff
@@ -1,6 +1,13 @@
-<svg viewBox="0 0 15 15" >
-	<g id="Layer_1_1_" class="view-size-svg">
-		<rect x="0" y="0" width="15" height="7"/>
-		<rect x="0" y="8" width="15" height="7"/>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we want the extra tag in these? I don't know enough about `<svg>` to know if that's good or not.